app.js

```diff
@@ -0,0 +1,44 @@
+const express = require("express");
+const app = express();
+
+const { getStatus, getCategories } = require("./controller");
+
+app.use(express.json());
+
+app.get("/api", getStatus);
```

Comment:
We don't actually need a getStatus path and controller, so this can be removed. The individual endpoints will respond with a 200 if everything's ok 👍